frontend/src/citizen-frontend/calendar/WeekElem.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import colors from 'lib-customizations/common'
+import { DailyReservationData } from './api'
+import DayElem from './DayElem'
+import { useTranslation } from '../localization'
+
+export interface WeekProps {
+  weekNumber: number
+  data: DailyReservationData[]
```

Comment:
I would prefer a more descriptive name like `dailyReservations` or something like that.